### DISCUSSION TITLE
Ways to unit test new azure durable orchestrators and activities

### DISCUSSION THREAD

**Discussion by vdboots** [Azure/azure-functions-durable-extension#2451] [Q&A]
Are there any examples available how to unit  test the new worker.extensions.durableTask. Extension orchestrators and activities

**Comment by cgillum** [comment]
I don't think we have examples for worker.extensions.durabletask yet (FYI @jviau). 

The Dapr Workflow project, which wraps the Durable Task .NET programming model, does have a small example that you could use as a reference: https://github.com/dapr/dotnet-sdk/blob/master/examples/Workflow/WorkflowUnitTest/OrderProcessingTests.cs. However, it assumes that you're using the class-based programming model.
